test/check-machines-nics

```diff
@@ -121,8 +121,13 @@ class TestMachinesNICs(machineslib.VirtualMachinesCase):
         # dropdown options are not visible, but still part of DOM tree, so let's use _wait_present
         b._wait_present("#vm-subVmTest1-add-iface-source option:nth-of-type(1):contains(0bridge)")
         b._wait_present("#vm-subVmTest1-add-iface-source option:nth-of-type(2):contains(abridge)")
-        b._wait_present("#vm-subVmTest1-add-iface-source option:nth-of-type(3):contains(virbr0)")
-        b._wait_present("#vm-subVmTest1-add-iface-source option:nth-of-type(4):contains(xbridge)")
+        if m.ws_container:
+            b._wait_present("#vm-subVmTest1-add-iface-source option:nth-of-type(3):contains(cni-podman0)")
```

Comment:
Interesting (and completely unrelated) -- could I actually put a libvirt VM and a container into the same virtual network that way?

Reply:
Uhmm I am not sure!